src/pages/manage/about.tsx

```diff
@@ -319,3 +170,119 @@ const useStyles = createStyles(() => ({
+        cache: 'force-cache',
+      }).then((res) => res.json()),
+  });
+  const { attributes } = usePackageAttributesStore();
+  if (!i18n) {
+    return;
+  }
+  const initOptions = i18n.options as ExtendedInitOptions;
+
+  const newVersionAvailable =
+    data?.tag_name > `v${attributes.packageVersion}` ? data?.tag_name : undefined;
+  const informations = useInformationTableItems(newVersionAvailable);
+  const { t } = useTranslation(['layout/modals/about']);
+  const { classes } = useStyles();
+
+  const keybinds = [
+    { key: 'Mod + J', shortcut: t('layout/modals/about:actions.toggleTheme') },
+    { key: 'Mod + K', shortcut: t('layout/modals/about:actions.focusSearchBar') },
+    { key: 'Mod + B', shortcut: t('layout/modals/about:actions.openDocker') },
+    { key: 'Mod + E', shortcut: t('layout/modals/about:actions.toggleEdit') },
+  ];
+  const rows = keybinds.map((element) => (
+    <tr key={element.key}>
+      <td>
+        <Kbd>{element.key}</Kbd>
+      </td>
+      <td>
+        <Text>{element.shortcut}</Text>
+      </td>
+    </tr>
+  ));
+
+  return (
+    <ManageLayout>
+      <Stack>
+        <Text>
+          <Trans i18nKey="layout/modals/about:description" />
+        </Text>
+
+        <Table withBorder>
+          <tbody>
+            {informations.map((item, index) => (
+              <tr key={index}>
+                <td>
+                  <Group spacing="xs">
+                    <ActionIcon className={classes.informationIcon} variant="default">
+                      {item.icon}
+                    </ActionIcon>
+                    <Text>
+                      <Trans
+                        i18nKey={`layout/modals/about:metrics.${item.label}`}
+                        components={{ b: <b /> }}
+                      />
+                    </Text>
+                  </Group>
+                </td>
+                <td className={classes.informationTableColumn} style={{ maxWidth: 200 }}>
+                  {item.content}
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </Table>
+        <Accordion mb={5} variant="contained" radius="md">
+          <Accordion.Item value="keybinds">
+            <Accordion.Control icon={<IconKey size={20} />}>
+              {t('layout/modals/about:keybinds')}
+            </Accordion.Control>
+            <Accordion.Panel>
+              <Table mb={5}>
+                <thead>
+                  <tr>
+                    <th>{t('layout/modals/about:key')}</th>
+                    <th>{t('layout/modals/about:action')}</th>
+                  </tr>
+                </thead>
+                <tbody>{rows}</tbody>
+              </Table>
+              <Tip>{t('layout/modals/about:tip')}</Tip>
+            </Accordion.Panel>
+          </Accordion.Item>
+        </Accordion>
+
+        <TranslatorsTable loadedLanguages={initOptions.locales.length} />
+        <Divider />
+        <ContributorsTable contributors={contributors} />
```

Comment:
Switch order of translators and contributors

Reply:
Nope, translator has more info it's better to have it first